src/services/jobs.js

```diff
@@ -3,185 +3,186 @@
-    }
-};
+ * create new job instance that exicute repeatly at given interval
+ * @method
+ * @name jobInstanceForEvery
+ * @param {Object} jobName -name of job definition.
+ * @param {Object} scheduleInterval -job exicution interval.
+ * @returns {JSON} - returns job instance creation details.
+ */
+
+const createJobInstanceForEvery = async (jobName, scheduleInterval) => {
+	try {
+		const job = {
+			name: jobName,
+			interval: scheduleInterval,
+		}
+		//get all jobs definition from DB
+		const jobs = await jobsReady
+		//checking job Definition is present
+		const jobExists = await utils.checkForDuplicateJobDefinition(job, jobs)
+
+		if (jobExists > 0) {
+			const scheduleJob = await scheduleEvery(job, agenda)
+			return scheduleJob
+		} else {
+			return {
+				message: responseMessage.JOB_DEFINITION_NOT_FOUND,
+				success: false,
+				status: httpResponse.BAD_REQUEST,
+			}
+		}
+	} catch (err) {
+		return {
+			message: err,
+			success: false,
+			status: httpResponse.BAD_REQUEST,
+		}
+	}
+}
 
 /**
-    * create new job instance that exicute immediately.
-    * @method
-    * @name jobInstanceForNow
-    * @param {Object} jobName -name of job definition to be exicuted.
-    * @returns {JSON} - returns job instance creation details.
-*/
-
-const createJobInstanceForNow = async ( jobName ) => {
-    try {
-        const job = {
-            name : jobName
-        };
-        //get all jobs definition from DB
-        const jobs = await jobsReady;
-        //checking job Definition is present
-        const jobExists = await utils.checkForDuplicateJobDefinition( job, jobs );
-
-        if ( jobExists > 0 ) {
-            const scheduleJob = await scheduleNow( jobName, agenda );
-            return scheduleJob;
-        } else {
-            return ({
-                message : responseMessage.JOB_DEFINITION_NOT_FOUND,
-                success : false,
-                status : httpResponse.BAD_REQUEST
-            });
-        }
-    }catch(err){
-        return ({
-                message : err,
-                success : false,
-                status : httpResponse.BAD_REQUEST
-            });
-    }
-};
+ * create new job instance that exicute immediately.
+ * @method
+ * @name jobInstanceForNow
+ * @param {Object} jobName -name of job definition to be exicuted.
+ * @returns {JSON} - returns job instance creation details.
+ */
+
+const createJobInstanceForNow = async (jobName) => {
+	try {
+		const job = {
+			name: jobName,
+		}
+		//get all jobs definition from DB
+		const jobs = await jobsReady
+		//checking job Definition is present
+		const jobExists = await utils.checkForDuplicateJobDefinition(job, jobs)
+
+		if (jobExists > 0) {
+			const scheduleJob = await scheduleNow(jobName, agenda)
+			return scheduleJob
+		} else {
+			return {
+				message: responseMessage.JOB_DEFINITION_NOT_FOUND,
+				success: false,
+				status: httpResponse.BAD_REQUEST,
+			}
+		}
+	} catch (err) {
+		return {
+			message: err,
+			success: false,
```

Comment:
@VISHNUDAS-tunerlabs failure response should also come from common function, change in all the places